### PR TITLE
NextVersionLoadingAnimation

### PR DESCRIPTION
**Pull request by jjaejjae12**
* change react -> next
* add LoadingAnimation module

### CODE REVIEW

File: src/components/LoadingAnimation/style.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+export const Loader = styled.div`
+  position: relative;
+  width: 200px;
+  height: 200px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const Sapn = styled.span`
```
Comment (kangkong0805) [inline src/components/LoadingAnimation/style.ts]: 음...?

File: src/components/LoadingAnimation/index.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import * as S from "./style";
+
+export default function LoadingAnimation() {
+  return (
+    <S.bg>
+      <S.Loader>
+        <S.Sapn></S.Sapn>
```
Comment (kangkong0805) [inline src/components/LoadingAnimation/index.tsx]: 이거 먼가요...?

**Review by kangkong0805 — CHANGES_REQUESTED** [review]

File: src/components/LoadingAnimation/style.ts (hunk 1 above)
Comment (jjaejjae12) [reply src/components/LoadingAnimation/style.ts]: 스타일 컴포넌트 변수명 [수정](https://github.com/Frontend-Design/Design/pull/8/commits/3e72cea8a6b54b4ceea751b2ae89bdacf370907c)

File: src/components/LoadingAnimation/style.ts [hunk 3]
```diff
@@ -17,7 +17,7 @@ export const Loader = styled.div`
   align-items: center;
 `;
 
-export const Sapn = styled.span`
+export const Span = styled.span`
```
Comment (kangkong0805) [inline src/components/LoadingAnimation/style.ts:20]: ㄴㅇㅅ

File: src/components/LoadingAnimation/index.tsx (hunk 2 above)
Comment (jjaejjae12) [reply src/components/LoadingAnimation/index.tsx]: tsx 태그명 [수정](https://github.com/Frontend-Design/Design/pull/8/commits/3e72cea8a6b54b4ceea751b2ae89bdacf370907c)

File: src/components/LoadingAnimation/style.ts (hunk 3 above)
Comment (jjaejjae12) [reply src/components/LoadingAnimation/style.ts:20]: merge ㄱ?

**Review by kangkong0805 — APPROVED** [review]

**Pull request merged**